package.json

```diff
@@ -247,6 +252,16 @@
                 "command": "githd.viewStashes",
                 "title": "View Stashes",
                 "category": "GitHD"
+            },
+            {
+                "command": "githd.revealCommittedFileInEditor",
```

Comment:
We don't need this. The functionality is there. 
![Screenshot 2024-04-14 at 9 28 23 PM](https://github.com/huizhougit/githd/assets/4254119/78fce715-c9ff-4fdd-8b07-c0df38f028fc)

Reply:
It may be more convenient to add this function to the context menu as well because the built-in "Open File" is too far away from the explorer view.